dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/SubmissionService.java

```diff
@@ -265,7 +265,7 @@ public XmlWorkflowItem createWorkflowItem(Context context, String requestUriList
             wi = workflowService.start(context, wsi);
         } catch (IOException e) {
             throw new RuntimeException("The workflow could not be started for workspaceItem with" +
-                                               "id:  " + id);
+                                               "id:  " + id, e);
```

Comment:
Tiny issue:  there's a space missing between "with" and "id".

Reply:
@mwoodiupui : Good catch, I missed that in the initial code.  Fixed.